### PR TITLE
Pro issue 4302

### PR DESCRIPTION
**Pull request by Liviu-p**

### CODE REVIEW

File: js/formidable_admin.js [hunk 1]
```diff
@@ -7697,8 +7698,21 @@ function frmAdminBuildJS() {
 		}
 	}
 
+	function reloadIfAddonActivatedAjaxSubmitOnly() {
```
Comment (Crabcyborg) [inline js/formidable_admin.js:7701]: This isn't part of this update. Branches should be made from master, not from other unmerged branches.

It makes it more difficult to review and there can be unintended mistakes.

File: js/formidable.js [hunk 2]
```diff
@@ -1882,3 +1882,10 @@ function frm_resend_email( entryId, formId ) { // eslint-disable-line camelcase
 		}
 	});
 }
+
+( function() {
+	// Elementor popup show event. Fix Elementor Popup && FF Captcha field conflicts
+	jQuery( document ).on( 'elementor/popup/show', () => {
```
Comment (truongwp) [inline js/formidable.js:1888]: Can we move this to inside of `frmFrontFormJS()`? We don't need an extra `( function() {}() )` here.

Comment (Liviu-p) [reply js/formidable.js]: sure thing @truongwp, I've just updated it.

**Review by Crabcyborg — APPROVED** [review]
I just changed the `frmCaptcha` function back to what it was. I don't think this change was necessary.

Trying to initialize all `.g-recaptcha` could possibly cause conflicts other other captchas. Only checking the `frm-` prefixed class is safer.

I didn't notice any difference when changing this back.

I also reduced the code a bit. I removed the arrow function. It's down to a single line update which is nice.

Thanks for working on this @Liviu-p!

And thank you @truongwp for your help reviewing it as well.

**Pull request merged**